### ISSUE TITLE
Batch remove deleted branches ("Cleanup")

### ISSUE THREAD

**Issue by dewey** [ForkIssues/Tracker#2200]
Hey, Fork has this nice feature where it shows when a branch was deleted (Because it was merged on GitHub). It would be nice if there's a way to clean them all at once instead of having to find all of them and then right click -> "Delete n branches".

Maybe in the "Repository" -> "Remove deleted branches" could make sense?

What do you think? Is there already a better way?

Thanks!

![Screenshot 2024-09-06 at 11 25 42 png@2x](https://github.com/user-attachments/assets/d7d3a5cf-fefe-474d-bef5-b85123f2fca7)

![Screenshot 2024-09-06 at 11 26 54 png@2x](https://github.com/user-attachments/assets/c27943c3-427c-4126-a5ac-f8a1d63da38d)

**Comment by DanPristupov** [comment]
> It would be nice if there's a way to clean them all at once instead of having to find all of them and then right click -> "Delete n branches".

Selecting them manually and then deleting at once is the correct way as this reduces the chance of user error.

https://github.com/fork-dev/Tracker/issues/1759#issuecomment-1342278433
https://github.com/fork-dev/Tracker/issues/1605#issuecomment-1129587907
https://github.com/fork-dev/Tracker/issues/1716
https://github.com/fork-dev/Tracker/issues/1414

**Comment by dewey** [comment]
Huh, I searched the issues but was somehow not able to find these. Thanks!

I'd argue that I'm more likely to delete a branch accidentally by selecting the wrong row after doing a manual multi select in a long list of open / deleted branches (As they are sorted alphabetically and not by "status" like deleted or active) vs catching all of them programmatically and maybe having a confirmation modal listing them all.

**Comment by wrymn** [comment]
Indeed, this request is quite valid, and user error increases as Im manually must select all inactive branches, while some branch I dont wanna delete is the one in selection

**Comment by matthewjamesadam** [comment]
This would be a great feature -- having to clean this list up every couple days is a chore.

**Comment by luccasmaso** [comment]
Totally agree, would be super useful!

**Comment by DanPristupov** [comment]
In Fork 2.62, we added an option to select all local branches whose remote tracking branches no longer exist. You can then delete them all at once.
<img width="295" height="285" alt="Image" src="https://github.com/user-attachments/assets/50110e02-d9c9-48bd-8408-40973f101e2b" />

**Comment by dewey** [comment]
That's perfect, thanks @DanPristupov! 🎉
